src/test/java/seedu/address/AppParametersTest.java

```diff
@@ -37,6 +38,19 @@ public void parse_invalidConfigPath_success() {
         assertEquals(expected, AppParameters.parse(parametersStub));
     }
 
+    @Test
+    public void hashCode_test() {
+        AppParameters duplicateParams = new AppParameters();
+        AppParameters differentParams = new AppParameters();
+
+        expected.setConfigPath(Paths.get("config.json"));
+        duplicateParams.setConfigPath(Paths.get("config.json"));
+        differentParams.setConfigPath(Paths.get("another.json"));
+
+        assertEquals(duplicateParams.hashCode(), expected.hashCode());
+        assertNotEquals(differentParams.hashCode(), expected.hashCode());
+    }
+
```

Comment:
Can test the equals method, which currently isnt fully tested, can try .equals(NOT A APPPARAMETER TYPE), .equals(the same params)